dvc/cli.py

```diff
@@ -145,6 +145,13 @@ def get_parent_parser():
         "-v", "--verbose", action="count", default=0, help="Be verbose."
     )
 
+    parent_parser.add_argument(
+        "--profile",
```

Comment:
A bit worried about this colliding with `profile` from s3 and other similar things in the future.

Reply:
Maybe `--cprofile` ?

Reply:
Yup, could do. How about just `--prof`?

Reply:
@skshetry IIRC, argparse counts `--prof` as `--profile` if you have it, so that won't work :D